desc/compute/_omnigenity.py

```diff
@@ -328,7 +385,83 @@ def _sqrtg_B(params, transforms, profiles, data, **kwargs):
 
 
 @register_compute_fun(
-    name="|B|_mn",
+    name="sqrt(g)_B",
```

Comment:
I suggest naming this sqrt(g)_Boozer. At least something different than the old name. Otherwise this PR redefines sqrt(g)_B and this may affect external code